### DISCUSSION TITLE
系统是centos7.4，打开文件背景颜色是天蓝色，请问有什么办法可以解决？看的我眼睛好花

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#824] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/110797126/4b32c356-8549-4a54-aac2-57095985d594)
如上图所示

**Comment by ayamir** [comment]
什么终端？什么终端背景？用的是什么主题？什么信息都没有我无法提供帮助。

**Reply by wxzvim** [reply]
xshell终端，Xterm配色方案，用的下载neovim默认主题，还有发现一件事，，就是图标乱码，，怎么解决？也是xshell终端，，用的DejaVu Sans Mono字体，，我windows安装了DejaVuSansMNerdFontMono-Regular.ttf字体，，包含了nerd fonts补丁，，为啥还乱码，，还是说在centos7上安装相应的字体，，xshell显示才不乱吗？

**Reply by Jint-lzxy** [reply, marked as answer]
> xshell终端

你使用的终端机(Xshell)需要使用者自行启用「支援真彩色」这个选项，引述termstandard/colors的[README](https://github.com/termstandard/colors/blob/master/README.md)：
> - [Netsarang XShell](https://www.netsarang.com/products/xsh_overview.html) -
>  Xshell7/ Xshell6 >= Build 0181
 > (You must set _**Tools-Options.. -Advanced**_, check the _**Use true color\***_ and **reopen** the software)
***
> 还是说在centos7上安装相应的字体，，xshell显示才不乱吗？

你是不是有在用SSH？这样的话你需要确认Xshell的设定中[使用了任一Nerd Font作为它的首选字型](https://netsarang.atlassian.net/wiki/spaces/ENSUP/pages/419957078/Font+Change)：
> In **Xshell**, only fixed width fonts may be selected.                                                                                                                                                                               
> **To Change Terminal Font and the Font Size:**
> 1. Do one of the following:
>   - Click the standard [Fonts] button and select a font and the font size.
>   - Select [Appearance] in **Session Properties** dialog box and select a font and the font size.
>
> **To Change Bold Text:**
> 1. Open the **Session Properties** dialog box.
> 2. Select [Appearance] from [Category].
> 3. Select an option from [Bold] field.

**Reply by wxzvim** [reply]
牛逼plus，感谢大佬，改为工具-窗口-高级-使用本色解决呢，感谢感谢，真想请你喝杯

**Reply by wxzvim** [reply]
感谢，第二个问题也解决呢，我安装了DejaVuSansM Nerd Font Mono常规字体，原来得重启下windows机器，才能在xshell中生效，xshell选择这个字体，，图标就好了

**Reply by wxzvim** [reply]
另外；tmux开启也是天蓝色，，也是因为要开启真彩色，参考链接为
[帮助链接](https://lotabout.me/2018/true-color-for-tmux-and-vim/)

**Comment by wxzvim** [comment]
> 什么终端？什么终端背景？用的是什么主题？什么信息都没有我无法提供帮助。

xshell终端，Xterm配色方案，用的下载neovim默认主题，还有发现一件事，，就是图标乱码，，怎么解决？也是xshell终端，，用的DejaVu Sans Mono字体，，我windows安装了DejaVuSansMNerdFontMono-Regular.ttf字体，，包含了nerd fonts补丁，，为啥还乱码，，还是说在centos7上安装相应的字体，，xshell显示才不乱吗？

**Comment by wxzvim** [comment]
> > xshell终端
> 
> 你使用的终端机(Xshell)需要使用者自行启用「支援真彩色」这个选项，引述termstandard/colors的[README](https://github.com/termstandard/colors/blob/master/README.md)：
> 
> > * [Netsarang XShell](https://www.netsarang.com/products/xsh_overview.html) -
> >   Xshell7/ Xshell6 >= Build 0181
> >   (You must set _**Tools-Options.. -Advanced**_, check the _**Use true color***_ and **reopen** the software)
> 
> > 还是说在centos7上安装相应的字体，，xshell显示才不乱吗？
> 
> 你是不是有在用SSH？这样的话你需要确认Xshell的设定中[使用了任一Nerd Font作为它的首选字型](https://netsarang.atlassian.net/wiki/spaces/ENSUP/pages/419957078/Font+Change)：
> 
> > In **Xshell**, only fixed width fonts may be selected.
> > **To Change Terminal Font and the Font Size:**
> > 
> > 1. Do one of the following:
> > 
> > 
> > * Click the standard [Fonts] button and select a font and the font size.
> > * Select [Appearance] in **Session Properties** dialog box and select a font and the font size.
> > 
> > **To Change Bold Text:**
> > 
> > 1. Open the **Session Properties** dialog box.
> > 2. Select [Appearance] from [Category].
> > 3. Select an option from [Bold] field.

**Comment by wxzvim** [comment]
奇怪，这个问题怎么关闭了，如何重新打开？

**Reply by Jint-lzxy** [reply]
This discussion hasn't been closed yet. Do u have any further questions?

**Comment by wxzvim** [comment]
怎样打开此问题？不小心点错了，，关闭了问题

**Comment by Jint-lzxy** [comment]
Closed due to inactivity. Please reactivate if further support is needed.
